paymentsheet/src/test/java/com/stripe/android/paymentsheet/state/DefaultPaymentSheetLoaderTest.kt

```diff
@@ -977,6 +977,50 @@ internal class DefaultPaymentSheetLoaderTest {
         assertThat(result.linkState).isNull()
     }
 
+    @Test
+    fun `When EPMs are requested but not returned by elements session, no EPMs are used`() = runTest {
+        val requestedExternalPaymentMethods = listOf("external_paypal")
```

Comment:
Is it worth using named params in the call to `testExternalPaymentMethods` instead of having separate variables?

Reply:
sgtm! Updated the PR